experimental/cloudstrapper/playbooks/roles/aws-cleanup/tasks/orc8r-ec2-asg.yaml

```diff
@@ -0,0 +1,22 @@
+---
+
+- name: list autoscaling groups
+  ec2_asg_info:
+    tags:
+      Name: "{{ orc8rAsgClusterTagName }}"
+  register: valAsgCluster
+
+- name: debug
+  debug:
+    msg: "{{ valAsgCluster }}"
+
+- name: set autoscaling group fact
```

Comment:
curious, why set this fact instead of directly valAsgCluster.results[0].auto_scaling_group_name ? Is it just for clarity?

Reply:
Yes - In the normal case, there will really be one ASG, so it should be okay to directly use the value at [0]. However, if there are more ASGs, we can filter them based on tags and select just that one.